### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'getScope')

### ISSUE THREAD

**Issue by mrstork** [mthadley/eslint-plugin-sort-destructure-keys#277]
#### Error message:
```
TypeError: Cannot read properties of undefined (reading 'getScope')
Occurred while linting [...].js:25
Rule: "sort-destructure-keys/sort-destructure-keys"
at ObjectPattern (/[...]/node_modules/eslint-plugin-sort-destructure-keys/lib/rules/sort-destructure-keys.js:220:34)
...
```

#### Versions:
- eslint-plugin-sort-destructure-keys: 1.6.0
- ESLint: 8.20.0

#### Notes:

The issue seems to be coming from the following code block:
```
const scope = sourceCode.getScope ? sourceCode.getScope(objectPatternNode) : context.getScope();
```
There looks to be a patch in `master` that ensures `sourceCode` is defined before calling `getScope` on it, however this change isn't in the 1.6.0 tag. Would we be able to include it in a v1.6.1 release?

**Comment by mthadley** [comment]
@mrstork Are you able to install [`v2.0.0`](https://www.npmjs.com/package/eslint-plugin-sort-destructure-keys/v/2.0.0)?

**Comment by mrstork** [comment]
Sorry about this, I see this is also resolved in the latest version `2.0.0` which is sufficient for me.

**Comment by mthadley** [comment]
No problem! Glad it is working for you.

Closing since if other people have this issue, I'll have the same recommendation of upgrading to `v2.0.0`.
